grabbit/src/main/groovy/com/twcable/grabbit/client/batch/steps/validation/ValidJobDecider.groovy

```diff
@@ -0,0 +1,82 @@
+package com.twcable.grabbit.client.batch.steps.validation
+
+import com.twcable.grabbit.client.batch.ClientBatchJob
+import com.twcable.grabbit.client.batch.ClientBatchJobContext
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+import org.springframework.batch.core.JobExecution
+import org.springframework.batch.core.StepExecution
+import org.springframework.batch.core.job.flow.FlowExecutionStatus
+import org.springframework.batch.core.job.flow.JobExecutionDecider
+
+import javax.jcr.PathNotFoundException
+import javax.jcr.RepositoryException
+import javax.jcr.Session
+
+/*
+ * Copyright 2015 Time Warner Cable, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@CompileStatic
+@Slf4j
+class ValidJobDecider implements JobExecutionDecider {
+
+
```

Comment:
Cosmetic: Instead of two blank lines after the class definition, how about one blank line after the class definition and one blank line before class termination.